crates/control-interface/src/lib.rs

```diff
@@ -668,7 +668,7 @@ pub async fn collect_sub_timeout<T: DeserializeOwned>(
                     }
                 }
             },
-            _ = &mut sleep => { /* timeout */ break; }
+            () = &mut sleep => { /* timeout */ break; }
```

Comment:
What command did you run/what's your clippy setup look like? I haven't seen clippy complain about or fix this

Reply:
This is a [new lint called `ignored_unit_patterns` on Rust 1.73](https://github.com/rust-lang/rust-clippy/blob/master/CHANGELOG.md#rust-173) ([Clippy Lints](https://rust-lang.github.io/rust-clippy/master/index.html#/ignored_unit_patterns))